templates/kong-elb-dbless-vpc-optional-hvm.template

```diff
@@ -472,8 +472,8 @@
                 "yum install -y kong$kong_version\n",
                 "if [ \"$?\" -ne \"0\" ]\n",
                 "then\n",
-                "   /opt/aws/bin/cfn-signal -e 1 --stack ", { "Ref": "AWS::StackName" }, " --resource KongScalingGroup "," --region ", { "Ref" : "AWS::Region" }, " --reason \"failed to install kong$kong_version.aws.rpm\" \n",
-                "   echo \"[kong] failed to install kong$kong_version.aws.rpm, exiting...\" \n",
+                "   /opt/aws/bin/cfn-signal -e 1 --stack ", { "Ref": "AWS::StackName" }, " --resource KongScalingGroup "," --region ", { "Ref" : "AWS::Region" }, " --reason \"failed to install kong$kong_version.aws.amd64.rpm\" \n",
```

Comment:
shouldn't you updating https://github.com/Kong/kong-dist-cloudformation/pull/58/files#diff-7119ec232f3250d919f81e8c9a552225L460, the kong version we installing?

Reply:
oh this doesn't not operate the way I had thought when I made the PR 🤔 

Since it's not `curl / wget` the file and then installing the file this PR might be entirely unnecessary as `yum` _should_ grab the correct `rpm` from the yum repository